set.mm

```diff
@@ -7233,12 +7233,41 @@ _praeclarum theorema_ (splendid theorem).  (Contributed by NM,
 
 $( Relation of conjunction to bi-conditional. $)
 
-  ${
-    biadan2.1 $e |- ( ph -> ps ) $.
-    biadan2.2 $e |- ( ps -> ( ph <-> ch ) ) $.
-    $( Add a conjunction to an equivalence.  (Contributed by Jeff Madsen,
-       20-Jun-2011.) $)
-    biadan2 $p |- ( ph <-> ( ps /\ ch ) ) $=
+  $( An implication is equivalent to the equivalence of some implied
+     equivalence and some other equivalence.  A utility lemma as illustrated in
+     ~ biadanii and ~ elelb .  (Contributed by BJ, 4-Mar-2023.) $)
+  biadan $p |- ( ( ph -> ps ) <->
```

Comment:
The label does not fit to the description anymore. Formerly it was

> Add ("ad") a conjunction ("an") to an equivalence ("bi").

Reply:
Do you have a proposal ? (not too long)

Reply:
Maybe to enhance thew comment would be the simplest way to solve this issue:

> An implication is equivalent to the equivalence of some implied equivalence and an equivalence with a conjunction added.

Reply:
The label should depend only on the statement, so a change in the comment should not affect it. I had begun writing a more precise comment, but it was basically paraphrasing the statement, adding nothing to it, so I settled to a shorter one. I'll see what I can do.

Reply:
Done:
An implication is equivalent to the equivalence of some implied equivalence and some other equivalence involving a conjunction.